libs/blocks/merch/merch.css

```diff
@@ -8,6 +8,8 @@ span[data-wcs-osi] {
 }
 
 span.placeholder-resolved[data-template="strikethrough"], span.price.price-strikethrough {
+  font-size: 14px;
```

Comment:
Same comment as a previous PR, could we use `--type-body-xs-size` instead to also account for Arabic, which has special font rules?